boards/arm/nucleo_h745zi_q/nucleo_h745zi_q.dtsi

```diff
@@ -28,3 +28,12 @@
 		};
 	};
 };
+
+&rcc {
```

Comment:
we could retrieve the comment like the in the commit boards/arm/nucleo_h723zg/nucleo_h723zg.dts   does, for example
`hpre = <2>;    /* HCLK: 240   MHz */`

Reply:
Sure